docs/adr/0002-google-analytics-4-tracking-and-code-location.md

```diff
@@ -0,0 +1,19 @@
+# 2. Google Analytics 4 tracking and code location
+
+Date: 2022-05-09
+
+## Status
+
+Accepted
```

Comment:
I wonder if this should start off being 'pending' or 'TBC' to start off with? I guess it will depend if we use the Github PR reviews for ADR comments or if we use a google doc and then the PR is more of a final version? It looks like the ADRs are also documented in the dev docs (for example [Github](https://github.com/alphagov/smart-answers/blob/main/docs/arch/003-testing-flows.md) and [dev docs](https://docs.publishing.service.gov.uk/repos/smart-answers/arch/003-testing-flows.html)) so I wonder if there should be a mention of also adding to the docs in the other general ADR file ([0001-record-architecture-decisions.md](https://github.com/alphagov/govuk_publishing_components/pull/2761/files#diff-e2c27172663e2ac0af76f95037853f6a52e5f77889e90ad1c1f8644279f74402)).

Reply:
Yeah, good point - I think you got to this PR before I'd had chance to create the Google doc that we've agreed to work in 🚤  But it's worth changing it anyway - now `In progress`. Thanks @Rosa-Fox